test/adb-specs.js

```diff
@@ -0,0 +1,123 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import { ADB } from '../lib/adb.js';
+
+const should = chai.should();
+chai.use(chaiAsPromised);
+
+describe('ADB', () => {
+  it('should correctly return adb if present', async () => {
+    let adb = new ADB();
+    let temp = await adb.createADB();
+    should.exist(temp.path);
+  });
+  it('should correctly return adb if present ANDROID_HOME is not present', async () => {
```

Comment:
oops yeah my bad .... thanks for pointing that out, my brain is reading english like code with no emphasis on grammer :p